.github/workflows/push_to_canary.yaml

```
@@ -74,12 +72,9 @@ jobs:
         run: |
           set -eo pipefail
           echo "Building wheel"
-          python3 -m pip install --upgrade pip
-          python3 -m pip install --upgrade setuptools wheel
-          
-          ./scripts/distribution/build_wheel.sh ${{ env.VERSION }}
+          ./mill python.wheel ${{ env.VERSION }}
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)